schemes/zhang_mcfarlane/zm_convr.meta

```diff
@@ -186,100 +193,115 @@
   dimensions = ()
   intent = in
 [ latice ]
-  standard_name = latent_heat_of_fusion_of_water_at_0c?
-  units = J kg-1?
+  standard_name = latent_heat_of_fusion_of_water_at_0c
+  units = J kg-1
   type = real | kind = kind_phys
   dimensions = ()
   intent = in
 [ cpwv ]
-  standard_name = specific_heat_of_water_vapor_at_constant_pressure?
+  standard_name = specific_heat_of_water_vapor_at_constant_pressure
   units = J kg-1 K-1
   type = real | kind = kind_phys
   dimensions = ()
   intent = in
 [ cpliq ]
-  standard_name = specific_heat_of_liquid_water_at_20c
+  standard_name = specific_heat_of_liquid_water_at_constant_pressure
   units = J kg-1 K-1
   type = real | kind = kind_phys
   dimensions = ()
   intent = in
 [ rh2o ]
-  standard_name = gas_constant_of_water_vapor?
-  units = J kg-1 K-1?
+  standard_name = gas_constant_of_water_vapor
+  units = J kg-1 K-1
   type = real | kind = kind_phys
   dimensions = ()
   intent = in
+[ lat ]
+  standard_name = latitude
+  units = rad
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
+[ long ]
+  standard_name = longitude
+  units = rad
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent)
+  intent = in
 [ t ]
   standard_name = air_temperature
   units = K
   type = real | kind = kind_phys
-  dimensions = (1:horizontal_loop_extent,1:vertical_layer_dimension)
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
   intent = in
 [ qh ]
   standard_name = water_vapor_mixing_ratio_wrt_moist_air_and_condensed_water
+  advected = true
   units = kg kg-1
   type = real | kind = kind_phys
-  dimensions = (1:horizontal_loop_extent,1:vertical_layer_dimension)
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
   intent = in
+  advected = true
```

Comment:
looks like this one has "advected = true" twice

Reply:
I guess I was over zealous.  Removed one